### ISSUE TITLE
 assert torch.isnan(output_voxels).sum().item() == 0

### ISSUE THREAD

**Issue by xioatian1** [JeffWang987/OpenOccupancy#58]
非常感谢你们开源了这个优秀的工作！
我尝试复现多模态baseline：Multimodal-R50_img1600_128x128x10.py
但遇到了图片所示问题，损失为零。我用的是两卡3090进行训练，并尝试将lr调整至3e-5，可是仍然报这个错误，请问您能提一些建议解决这个问题吗？感谢！

Thank you very much for open-sourcing this excellent work! 
I have attempted to reproduce the multimodal baseline: Multimodal-R50_img1600_128x128x10.py, but encountered the issue shown in the image, where the loss remains zero. I am using two 3090 GPUs for training and have tried adjusting the learning rate to 3e-5, but the error persists. Could you kindly provide some suggestions to resolve this issue? Thank you!

![image](https://github.com/user-attachments/assets/216c2be9-5bca-42fd-9b6b-a8547f51d26a)

**Comment by Wistrong** [comment]
你好，请问解决了吗？遇到了同样的问题
